src/views/User/ProfileUpdatePage/ProfileUpdatePage.jsx

```diff
@@ -10,28 +10,49 @@ import { Input } from '@/components/Input';
 import { FormError } from '@/components/FormError';
 import { USER_ERROR, REGEX } from '@/utils/constants';
 import { isValidInput } from '@/utils/helpers';
+import { GET, POST } from '@/apis/axios';
+import useSWR, { mutate } from 'swr';
 
 const ProfileUpdatePage = () => {
-  const [previewURL, setPreviewURL] = useState('');
+  const { data: userData } = useSWR('me', GET);
```

Comment:
```suggestion
  const { data: userData } = useSWR('/me', GET);
```

/ 안 써도 동작은 되지만 컨벤션으로 적어주세요! 밑에 axios들도 마찬가지입니다.